src/tokens/eth/0xB91C2a2b953D72f3EF890490669a0A41B0ADD5f7.json

```diff
@@ -0,0 +1,26 @@
+{
+  "symbol": "BEFX",
+  "name": "Belifex",
+  "type": "ERC20",
+  "address": "0xB91C2a2b953D72f3EF890490669a0A41B0ADD5f7",
+  "ens_address": "",
+  "decimals": 18,
```

Comment:
Decimal doesn't match what [etherscan shows](https://etherscan.io/token/0xb91c2a2b953d72f3ef890490669a0a41b0add5f7)

Reply:
Updated it, must have been a typo. Thank you for your time.